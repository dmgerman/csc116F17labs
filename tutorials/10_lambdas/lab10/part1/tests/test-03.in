4 8 15 16 23 42 -1
Victoria Vancouver Saanich Nanaimo