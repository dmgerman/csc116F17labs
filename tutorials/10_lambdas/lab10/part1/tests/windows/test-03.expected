Enter a sequence of integers, ending with -1: 
Vector VI: 4 8 15 16 23 42 
Enter a sequence of words: 
Vector VS: Victoria Vancouver Saanich Nanaimo 

print_vector_alternating: 4 15 23 

double_vector_elements: VictoriaVictoria VancouverVancouver SaanichSaanich NanaimoNanaimo 

negate_alternating: -4 8 -15 16 -23 42 

print_vector_offset (offset 6): 10 14 21 22 29 48 

print_vector_offset (offset 9): 13 17 24 25 32 51 

sum_vector(VI): 108
sum_vector(VS): VictoriaVancouverSaanichNanaimo

sum_vector_templated(VI): 108
sum_vector_templated(VS): VictoriaVancouverSaanichNanaimo

longest_string(VS): Vancouver

